src/api/chrome.ts

```diff
@@ -22,10 +23,10 @@ async function sendMessage(...args) {
             } else {
                 text = await response.text();
             }
-            messageListeners.forEach((cb) => cb({type: 'fetch-response', data: text, error: null, id}));
+            messageListeners.forEach((cb) => cb({type: MessageType.BACKGROUND_FETCH_RESPONSE, data: text, error: null, id}));
```

Comment:
Hmm, `BACKGROUND_FETCH_RESPONSE` is missing in this enum.

Reply:
Good catch. I'm alarmed that compiler didn't compalin during type checking. I fixed this and looked over all other changes to make sure everything is correct. 

While looking at it, I spotted the following on line 28:

https://github.com/darkreader/darkreader/blob/4ade4bb543b61b97bd8f1b0b68d10226fbc32812/src/api/chrome.ts#L28
Is this correct? I think instead of `err` it should be `error: err` or something like that because `Message` type does not have `err` but has `error`.

Reply:
I filed https://github.com/darkreader/darkreader/pull/6467 for that.